packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -183,54 +256,58 @@ export default class Meeting {
    * @memberof Meeting
    */
   join(options) {
-    // let {mediaDirection} = options;
-
-    const mediaDirection = {
-      sendAudio: true,
-      sendVideo: true,
-      receiveAudio: true,
-      receiveVideo: true,
-      pstn: false,
-      sendShare: false,
-      receiveShare: false
-    };
-    return this.joinMeeting()
-      .then(() =>
-        this.getLocalMedia({
+    const {mediaDirection} = options;
+    this.setCorrelationId(this.id);
```

Comment:
Is there a reason we need both `this.id`  and `this.correlationId` since they are equivalent? Will that change in the future

Reply:
yes